### DISCUSSION TITLE
Not to bad, but some things missing

### DISCUSSION THREAD

**Discussion by neilyoung** [PaddlePaddle/PaddleOCR#13082] [Q&A] [answered]
Was quickly testing PaddlOCR for EU number plate detection. Found this:

- In Germany it is vital to distinct between the city/county identifier and the rest of the plate content. Both are separated by different stickers on front and back plate

<img width="116" alt="image" src="https://github.com/PaddlePaddle/PaddleOCR/assets/731020/01244ec7-5600-44a9-89f2-074321a64ef1">

- It is essential to detect this separating stickers as such, otherwise recognitions - even though generally correct - can be ambiguous and there is no chance to correct that by any algorithm:

- PaddleOCR recognizes this plate correctly as ('B00288', 0.9805293083190918), but in case of "B[sticker]NO 1234" and "BN[sticker]O 1234" it would fail with the prediction "BNO1234", which is ambiguous because the first plate is a Berlin plate and the second a Bonn plate, which legally and technically could perfectly coexist.

How can PaddleOCR handle this? Is there a way to retrain the model(s) to tell them, how such stickers generally look like?

Also there seems to be a weakness w.r.t. German Umlauts. A Tübingen plate (TÜ) has been recognized as ('TuSN429', 0.9141650795936584) (at least the lower case character could be of help to filter the results by algorithm)

**Comment by GreatV** [comment, marked as answer]
To improve the accuracy of PaddleOCR for German license plates, you can create a custom dataset that includes images of plates with and without the separating stickers. Ensure this dataset covers various scenarios including different types of stickers and positions.

**Reply by neilyoung** [reply]
Thanks. Additional question: Is there any tutorial (english), which describes the steps for transfer learning necessary to perform for this particular scenario?

**Reply by GreatV** [reply]
Please refer to: https://github.com/PaddlePaddle/PaddleOCR/blob/main/doc/doc_en/training_en.md#3-data-and-vertical-scenes

**Reply by neilyoung** [reply]
OK, thanks. I understand. My question is: How could I specify my "stickers" in the label files? Is there any syntactical trick, which I could use to define the sticker, likes so?

![image](https://github.com/PaddlePaddle/PaddleOCR/assets/731020/7919049c-ee6a-4e5c-b98b-6095a5961c8c)

```
image_001.png, [country]AA[sticker]DE 4800
```

**Reply by GreatV** [reply]
This may not work, you need to use special characters for the different stickers.

**Reply by neilyoung** [reply]
You mean, I should use something, usually not present on license plates, like ":" or "+" or "!" or "?" ?

**Reply by GreatV** [reply]
Yes, you're right.
